core/src/shred.rs

```diff
@@ -1546,4 +1536,15 @@ mod tests {
             Err(reed_solomon_erasure::Error::TooFewShardsPresent)
         );
     }
+
+    #[test]
+    fn test_bench_shredder() {
```

Comment:
maybe remove this before merging this pr?

Reply:
It's ok to let it run.. It takes a few minutes, but could be useful for perf analysis

Reply:
Maybe as a bench then? Because in test form it doesn't provide any value unless analyzed manually?

Reply:
yeah, that makes sense. I can remove it from here for now.